docs/source/getting-started/index.rst

```diff
@@ -67,53 +68,57 @@ Now, use Yarn to install the project dependencies and build the app::
     yarn install
     yarn workspaces run build
 
-You should see ``Compiled successfully.`` in the output if everything is working as expected.
+You should see ``Compiled successfully.`` in the output if everything is worked expected.
 
 .. TODO: Give instructions for possible failures.
 
 We can now run the app in two steps.
 You'll need two terminal windows running for this.
-
 In one terminal, at the root of the ``create-daml-app`` directory, run the script::
 
     ./daml-start.sh
 
-This compiles the DAML component of the project and starts a *Sandbox* ledger for the app.
-The ledger in this case is stored in the Sandbox application memory; it is not persistent but is useful for testing and development.
-We'll leave the Sandbox running to serve requests from the UI, which result in changes to the in-memory ledger.
+This script is just shorthand for ``daml start`` with some arguments, which does a few things:
```

Comment:
Can we please burn this script with lots of :fire: instead of spending the effort to explain it?

Reply:
I remember there was discussion about having sandbox options in daml.yaml. Is this possible now? If not, do you suggest just showing the full command instead of the script? That doesn't seem super convenient.

Reply:
It’s not possible but it would be easy to add. I’m suggesting that we should prioritize implementing this instead of spending time on explaining workarounds.

Reply:
Understood. I will try to get to this, anyway will address in separate PRs

Reply:
Absolutely definitely for a separate PR! I’m also not saying that you have to be the one implementing this. I just don’t want us to spend time on explaining workarounds when we could be spending time on making them unnecessary instead.

Reply:
Makes sense, point taken. For now it's only one line of explanation - the 3 bullet points are about what `daml start` itself does.